client/src/main/java/org/evosuite/coverage/FitnessFunctions.java

```diff
@@ -282,6 +344,19 @@ public static TestSuiteFitnessFunction getFitnessFunction(Criterion criterion) {
 				return InputCoverageTestFitness.class;
 		case TRYCATCH:
 				return TryCatchCoverageTestFitness.class;
+		case VDDU: 
+		case VMDDU:                                                //modified harsh
+		case VCDDU:
+		case VCMDDU1:
+		case VCMDDU2:
+		case VRDDU:
+		case DDU_METHOD:
+		case DDU_METHOD_DTR:
+		case DDU_PUBLIC_METHOD:
+		case DDU_PUBLIC_METHOD_DTR:
+		case DDU_BRANCH:
+		case DDU_BRANCH_DTR:
+				return LineCoverageTestFitness.class;               //modified harsh  
```

Comment:
Remove `//modified harsh`.  And why all these cases return `LineCoverageTestFitness.class`?